grpc/lookup/v1/rls.proto

```diff
@@ -37,6 +37,8 @@ message RouteLookupRequest {
   string stale_header_data = 6;
   // Map of key values extracted via key builders for the gRPC or HTTP request.
   map<string, string> key_map = 4;
+  // Application-specific optional extensions.
+  repeated Any extensions = 5;
```

Comment:
These fields need to be of type "google.protobuf.Any" instead of just "Any".

Also, you'll need to add the following around line 19 above:

```
import "google/protobuf/any.proto";
```

And you'll need to add the appropriate dependency in the BUILD.bazel file.

Reply:
Thanks for the suggestions! Done.